org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/Source/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
This is a test folder, why it is changed?

Reply:
Please open an issue and assign it to me so that I/we don’t forget.

Reply:
See https://github.com/eclipse-jdt/eclipse.jdt.ui/issues/903